docs/gallery/domain/images.py

```diff
@@ -0,0 +1,244 @@
+#
+# :py:class:`~pynwb.image.OpticalSeries` is for series of images that were presented
+# to the subject as stimuli.
+# We will create an :py:class:`~pynwb.image.OpticalSeries` object with the name
+# ``"StimulusPresentation"`` representing what images were shown to the subject and at what times.
+#
+# Image data can be stored either in the HDF5 file or as an external image file.
+# For this tutorial, we will use fake image data with shape of (200, 50, 50, 3).
+# The second and third dimensions represent x and y.
+# The fourth dimension represents the RGB value (length of 3) for color images.
+#
+# NWB differentiates between acquired data and data that was presented as stimulus.
+# We can add it to the :py:class:`~pynwb.file.NWBFile` object as stimulus data using
+# the :py:meth:`~pynwb.file.NWBFile.add_stimulus` method.
+#
+
+image_data = np.random.randint(low=0, high=255, size=(200, 50, 50, 3), dtype=np.uint8)
+optical_series = OpticalSeries(
+    name="StimulusPresentation",  # required
+    distance=0.7,  # required
+    field_of_view=[0.2, 0.3, 0.7],  # required
+    orientation="lower left",  # required
+    data=image_data,
+    unit="n.a.",
+    format="raw",
+    starting_frame=[0.0],
+    rate=1.0,
+    comments="no comments",
+    description="The images presented to the subject as stimuli",
+)
+
+nwbfile.add_stimulus(timeseries=optical_series)
+
+####################
+# ImageSeries: Storing series of images as acquisition
+# ----------------------------------------------------
+#
+# :py:class:`~pynwb.image.ImageSeries` is a general container for series of images acquired during
+# the experiment. Image data can be stored either in the HDF5 file or as an external image file.
+# When color images are stored in the HDF5 file the color channel order is expected to be RGB.
+#
+# We can add raw data to the :py:class:`~pynwb.file.NWBFile` object as *acquisition* using
+# the :py:meth:`~pynwb.file.NWBFile.add_acquisition` method.
+#
+
+image_data = np.random.randint(low=0, high=255, size=(200, 50, 50, 3), dtype=np.uint8)
+behavior_images = ImageSeries(
+    name="ImageSeries",
+    data=image_data,
+    description="Image data of an animal moving in environment.",
+    unit="n.a.",
+    format="raw",
+    rate=1.0,
+)
+
+nwbfile.add_acquisition(behavior_images)
+
+####################
+# External files (e.g. video files of the behaving animal) can be added to the :py:class:`~pynwb.file.NWBFile` by creating
+# an :py:class:`~pynwb.image.ImageSeries` object using the ``external_file`` attribute that specifies the
+# path to the external file(s) on disk. The file(s) path must be relative to the path of the NWB file.
+# Either ``external_file`` or ``data`` must be specified, but not both.
+
+behavior_external_file = ImageSeries(
+    name="ExternalFiles",
+    description="Behavior video of animal moving in environment.",
+    unit="n.a.",
+    external_file=["/path/to/image/file.avi"],
+    format="external",
+    starting_frame=[0.0],
```

Comment:
```suggestion
    starting_frame=[0],
```

Starting frame should be an int